core/trino-main/src/main/java/io/trino/sql/analyzer/StatementAnalyzer.java

```diff
@@ -1228,11 +1228,30 @@ protected Scope visitTable(Table table, Optional<Scope> scope)
                 }
```

Comment:
Can you please drop this cleanup commit from this PR. I looks complicated and it is difficult for me that understand if you didn't change logic actually.

Reply:
This class is so ugly. And my code without makes this class even worse.